forge-gui/res/cardsfolder/upcoming/take_the_fall.txt

```diff
@@ -0,0 +1,10 @@
+Name:Take the Fall
+ManaCost:U
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | SubAbility$ DBBranch | StackDescription$ SpellDescription | SpellDescription$ Target creature gets -1/-0 until end of turn. It gets -4/-0 until end of turn instead if you control an outlaw. (Assassins, Mercenaries, Pirates, Rogues, and Warlocks are outlaws.) Draw a card.
```

Comment:
you could just do NumAtt$ X | IsCurse$ True here and skip the branch with
`SVar:X:Count$Compare Count$Valid Creature.Outlaw GE1.4.1`

Reply:
I'm afraid that `SVar:X` crashes the game in a similar way to my first attempt at the "committed a crime this turn" check did for Omenport Vigilante (as discussed on Discord). Here's the relevant files for convenience. Also I'd think, without word to the contrary, that the check should include tribal/kindred noncreatures.

[take_the_fall_B.txt](https://github.com/Card-Forge/forge/files/14860884/take_the_fall_B.txt)
[2024-04-04-00_takethefall.txt](https://github.com/Card-Forge/forge/files/14860887/2024-04-04-00_takethefall.txt)

Reply:
That variant worked well 👍 
Edited the file accordingly.

Reply:
like you said don't check for creature type in outlaw though